addons/mail/static/src/core/common/composer.js

```diff
@@ -262,53 +263,6 @@ export class Composer extends Component {
         }
     }
 
-    get CANCEL_OR_SAVE_EDIT_TEXT() {
```

Comment:
We cannot remove anything from a stable version. Mark it as deprecated instead, and remove it in master :)

Reply:
This also means translation will be lost and have to be re-done in all languages even though it's just a clean up and fixes nothing.

There is no good reason to change any of this in stable.

Reply:
There is a misunderstanding: I meant the `isSmall` branching of the getter can be removed, but not the whole getter.

Reply:
> This also means translation will be lost and have to be re-done in all languages even though it's just a clean up and fixes nothing.
> 
> There is no good reason to change any of this in stable.

It costs redo the translation, but the benefit is not having to override ci/security and much safer change, especially when there's diff there for the fix.

There is a reason, whether this is "good" or not is up to you, IMO simpler and safer code in that case is more of a benefit than having to re-do the translation here.

Reply:
And I don't have the previous diff anymore, but it's probably ok to change for the specific part that need to have diff on the PR, if having a full phrase with tags was not necessary for it (as it's the only goal of this method).

Reply:
> ci security doesn't need to be overridden, just have to use markup template as was changed in later versions

Oh I didn't know we had `markup` template syntax in 18.3, nor that it by-passes ci warnings. Could just that indeed for minimal diff.